transports/msmq/index.md

```diff
@@ -77,25 +77,24 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
+| Everyone       | Send                | Set by NServiceBus in Versions 6.0.x and below |
 
+NOTE: Starting with Versions 6.1.0 and above, the installers will not automatically grant permissions to the `Anonymous` and `Everyone` group. It will respect the existing queue permissions that have been set up for the endpoint. 
 
-When an endpoint sends a command or event message that source requires permission at the target queue. For endpoints using the either the publish-subscribe or request-response patterns, both parties actually send messages and must have permission to send messages to each other. This is to allow the Subscribe message to be sent from the subscribing endpoint to the publishing endpoint in case of publish-subscribe or the response to be send back to the client in case of request-response.
+Any endpoint that sends a message to a target endpoint requires the `Send` permission to be granted for the sending user account on the target queue. For example, if an `endpoint A` is running as `userA` and is sending a message to `endpoint B`, then `userA` requires the `Send` permission to be granted on `endpoint B`'s queue. When using messaging patterns like request-response or publish-subscribe, the queues for both the sending endpoint and the target endpoint will require `Send` permissions to be granted to each others user accounts. 
 
-NOTE: In versions 5.2.20 and 6.0.3 a bug was fixed where installers always changed permissions on a queue, also if it already existed.
-
-When an endpoint creates a queue on a machine, the default permissions depend on whether the machine is connected to a domain or a workgroup.
+When an endpoint creates a queue on a machine, the default permissions depend on whether the server is joined to a [domain or a workgroup](https://support.microsoft.com/en-us/help/884974/information-about-workgroup-mode-and-about-domain-mode-in-microsoft-me).
 
 
 ### Domain mode
 
-If the machine is connected to a domain, then only the domain user running the endpoint when the queue is created will have Send permissions granted. If multiple endpoints need to communicate and are running under the same domain account, no further configuration is required. If multiple endpoints need to communicate using different domain accounts then the Send permission on the receiving endpoint input queue needs to be granted to the domain account of the sending endpoint.
+If the machine is joined to a domain, then at the time of queue creation, only the domain user will have `Send` permissions granted. The `Everyone` user group will not have `Send` permissions. If all the endpoints which need to communicate are running under the same domain account, no further configuration is required. However, if the endpoints are run using different domain accounts, then the `Send` permission on the receiving endpoint's input queue needs to be explicitly granted to the domain user account of the sending endpoint.
```

Comment:
> The `Everyone` user group will not have `Send` permissions.

Why would I assume that it is? Just because of the table above? Should the table include some kind of indication?